packages/client/modules/userDashboard/components/OrgBilling/Billing.tsx

```diff
@@ -0,0 +1,335 @@
+})
+
+const Error = styled('div')<{isError: boolean}>(({isError}) => ({
+  alignItems: 'center',
+  color: isError ? PALETTE.TOMATO_500 : PALETTE.SLATE_600,
+  display: isError ? 'flex' : 'none',
+  lineHeight: '24px'
+}))
+
+const Message = styled('div')({
+  fontSize: 15,
+  paddingLeft: 4
+})
+
+const Billing = () => {
+  const [stripeClientManager] = useState(() => new StripeClientManager())
+  const {fields, onChange, setDirtyField} = useForm({
+    cardName: {
+      getDefault: () => '',
+      nomralize: stripeClientManager.normalizeCardName,
+      validate: stripeClientManager.validateCardName
+    },
+    cardNumber: {
+      getDefault: () => '',
+      normalize: stripeClientManager.normalizeCardNumber,
+      validate: stripeClientManager.validateCardNumber
+    },
+    cvc: {
+      getDefault: () => '',
+      normalize: stripeClientManager.normalizeCVC,
+      validate: stripeClientManager.validateCVC
+    },
+    expiry: {
+      getDefault: () => '',
+      normalize: stripeClientManager.normalizeExpiry,
+      validate: stripeClientManager.validateExpiry
+    }
+  })
+  const {cardName, cardNumber, cvc, expiry} = fields
+  const error =
+    // serverError || //TODO: add server error when adding functionality: https://github.com/ParabolInc/parabol/issues/7693
+    (cardNumber.dirty && cardNumber.error) ||
+    (expiry.dirty && expiry.error) ||
+    (cvc.dirty && cvc.error)
+  const canSubmit = !!(!error && cardName.value && cardNumber.value && expiry.value && cvc.value)
+
+  // TODO: add functionality in https://github.com/ParabolInc/parabol/issues/7693
+  // const handleSubmit = async (e: React.FormEvent) => {
+  //   e.preventDefault()
+  //   // if (submitting) return
+  //   // these 3 calls internally call dispatch (or setState), which are asynchronous in nature.
+  //   // To get the current value of `fields`, we have to wait for the component to rerender
+  //   // the useEffect hook above will continue the process if submitting === true
+
+  //   setDirtyField()
+  //   validateField()
+  //   submitMutation()
+  // }
+
+  return (
+    <StyledPanel label='Credit Card'>
+      <StyledRow>
+        <HeadingBlock>
+          <Heading>{'Your next payment is due on '}</Heading>
+          <Heading isBold>{' November 17, 2023.'}</Heading>
+        </HeadingBlock>
+      </StyledRow>
+      <StyledRow>
+        <Plan>
+          <Title>{'Credit Card Details'}</Title>
+          <Content>
+            <Form>
+              <InputLabel>{'Cardholder Name'}</InputLabel>
+              <StyledInput
+                autoFocus
+                maxLength={100}
+                placeholder='Full Name'
+                onBlur={() => setDirtyField('cardName')}
+                onChange={onChange}
+                id='cardholder-name'
+                name='cardName'
+                type='text'
+                autoComplete='cc-name'
+              />
+              <InputLabel>{'Card Number'}</InputLabel>
+              <CleaveInput
```

Comment:
should we use Stripe Elements to not risk being not PCI compliant?
https://github.com/ParabolInc/parabol/issues/4460#issuecomment-1281241237

Reply:
@nickoferrall small but nice detail, you can set Stripe Elements to use our font by passing `options` prop to `Elements` component

```
    <Elements options={{
      fonts: [
        {
          family: 'IBM Plex Sans',
          src: `url('/static/fonts/IBMPlexSans-Regular.woff2') format('woff2')`,
          weight: '400',
        },
      ],
     }}>
      ...
    </Elements>
```

Reply:
@nickoferrall this is nice! If it doesn't add too much bloat to override the styles to match the design, I'd like to have the customized bg and border. Thinking about form consistency across the app, we're sticking pretty close to the material design standard.

Other _small_ nitpick is the upgrade button. It should be the full width of the form, and disabled (grey) until the form is filled out